giraffe/src/components/geo/bing-maps/BingLayerObject.ts

```diff
@@ -0,0 +1,155 @@
+import L from 'leaflet'
+
+/*
+ * Based on https://github.com/TA-Geoforce/react-leaflet-bing-v2
+ */
+export const BingLayerObject = L.TileLayer.extend({
+  options: {
+    subdomains: [0, 1, 2, 3],
+    type: 'Aerial',
+    attribution: 'Bing',
+    culture: '',
+    style: '',
+  },
+
+  initialize: function(bing_key, options) {
+    L.Util.setOptions(this, options)
+
+    this._bing_key = bing_key
+    this._url = null
+    this._providers = []
+    this.metaRequested = false
+  },
+
+  tile2quad: function(x, y, z) {
+    let quad = ''
+    for (let i = z; i > 0; i--) {
+      let digit = 0
+      const mask = 1 << (i - 1)
+      if ((x & mask) !== 0) digit += 1
+      if ((y & mask) !== 0) digit += 2
```

Comment:
Please always use braces for if statements. We will be updating the eslint rules to reflect this.